assets/src/web-stories-block/block/icons.js

```diff
@@ -81,16 +86,185 @@ export const EMBED_STORY_BLOCK_ICON = (
   </SVG>
 );
 
+const Icon = styled.svg`
+  .is-pressed & circle {
+    stroke: black;
+  }
+`;
+
 // Icons for view types.
 export const CAROUSEL_VIEW_TYPE_ICON = (
-  /* From https://material.io/tools/icons */
+  <Icon
+    width="18"
+    height="14"
+    viewBox="0 0 18 14"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <rect width="10" height="14" transform="matrix(1 0 0 -1 4 14)" />
+    <rect width="2" height="10" transform="matrix(1 0 0 -1 16 12)" />
+    <rect width="2" height="10" transform="matrix(1 0 0 -1 0 12)" />
+  </Icon>
+);
+
+export const CIRCLES_VIEW_TYPE_ICON = (
+  <Icon
+    width="23"
+    height="18"
+    viewBox="0 0 23 18"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <Circle r="3" transform="matrix(1 0 0 -1 20 9)" />
+    <Circle r="3" transform="matrix(1 0 0 -1 3 9)" />
+    <Circle
+      r="7.1"
+      transform="matrix(1 0 0 -1 12 9)"
+      stroke="white"
+      strokeWidth="2.2"
+    />
+  </Icon>
```

Comment:
Not sure I understand hardcoding SVG files like this. Elsewhere just have the svg as a [file](https://github.com/rtCamp/web-stories-wp/tree/1795867f6456d3db7dd1cb6c127fe22554f7f512/assets/src/edit-story/icons) and [import](https://github.com/rtCamp/web-stories-wp/blob/1795867f6456d3db7dd1cb6c127fe22554f7f512/assets/src/edit-story/elements/video/icon.js#L25) it.

Reply:
our webpack build use `svgr/webpack`, which strips of the dimensions from the svg which are needed for these icons to be displayed on the block toolbar, without `width/height` the SVG spans full width and we don't have direct control via CSS either to limit the dimension without affecting other icons.

`svg/webpack` uses [config file](https://github.com/rtCamp/web-stories-wp/blob/feature/block-icons-update/.svgo.yml#L3), we wouldn't want to modify this here as it will affect the dashboard as well, so best option was to use them in JS as constants.

Reply:
I see what you mean. Could you please add a code comment in this file with this explanation? Perhaps with a TODO mark? It will make our lives as maintainers much easier.This way we can come back to it later and fix it.

----

Out of curiosity, have you tried something like this:

```js
import { ReactComponent as GridViewTypeIcon } from '../../icon.svg';
export const GRID_VIEW_TYPE_ICON = <GridViewTypeIcon width={12} height={14} />
```

(probably doesn't work, but worth giving it a shot)

Reply:
I haven't tried with the dimension attrs, though did try with the `size` attribute. I'll update the PR if it works.

Reply:
This worked, and I've updated the PR with the required changes.